pkgs/development/python-modules/dbus-signature-pyparsing/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyparsing
+, hypothesis
+, hs-dbus-signature
+}:
+
+buildPythonPackage rec {
+  pname = "dbus-signature-pyparsing";
+  version = "0.04";
+
+  src = fetchFromGitHub {
+    owner = "stratis-storage";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-IXyepfq7pLTRkTolKWsKGrYDoxukVC9JTrxS9xV7s2I=";
+  };
+
+  propagatedBuildInputs = [ pyparsing ];
+  checkInputs = [ hypothesis hs-dbus-signature ];
+
```

Comment:
It would be good to enforce testing by adding `ceckInputs = [ pytestCheckHook ]` and `pythonImprotsCheck = [ "..." ];` if possible